Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift

```diff
@@ -77,6 +77,17 @@ import BraintreeCore
     /// Optional. UI Customization for 3DS2 challenge views.
     public var v2UICustomization: BTThreeDSecureV2UICustomization?
 
+    /// Optional. Sets all UI types that the device supports for displaying specific challenge user interfaces in the 3D Secure challenge.
+    ///
+    /// Defaults to `.both`
+    public var uiType: BTThreeDSecureUIType = .both
+
+    /// Optional. List of all the render types that the device supports for displaying specific challenge user interfaces within the 3D Secure challenge.
+    ///
+    /// - Note: When using `BTThreeDSecureUIType.both` or `BTThreeDSecureUIType.html`, all `BTThreeDSecureRenderType` options must be set.
+    /// When using `BTThreeDSecureUIType.native`, all `BTThreeDSecureRenderType` options except `BTThreeDSecureRenderType.html` must be set.
+    public var renderTypes: [BTThreeDSecureRenderType.StringValue]?
```

Comment:
Arrays of `Int` enums do not exist in Obj-C so we cannot use the cleaner version (array of enums) here. 😢 This seems to be the consensus on the best practice in these cases. Open to other ideas if folks find something that works better as this is certainly not the most Swift-y approach. But working in the confines of Obj-C it seems to be our best bet.

Reply:
I don't have any better ideas for this but curious if we are planning to remove obj-c support in the next major version?

Reply:
Currently there isn't a plan to remove Obj-C support in the next major version but we could consider discussing it further as an option! Or if Apple announces an official deprecation policy at some point it'd be a much easier choice.